tests/runner/0x111256789012345678901234567890123456abef/README.md

```diff
@@ -0,0 +1,2 @@
+The files in this directory are used by the blockchain unit tests
+and are not part of the Scilla testsuite.
```

Comment:
Could we then extract these files into a separate directory? Looks like this change would interfere with my ongoing project of refactoring the scilla package into two packages. I'm experimenting with refactoring the test suite to use Dune for blackbox testing and the usual workflow there is directory-based.

Reply:
@anton-trunov I did the next best thing, added it to the testsuite.